src/pages/newCommunity/newCommunity.css

```diff
@@ -0,0 +1,32 @@
+.ncContent {
+  margin-top: 5%;
+  margin-left: 15%;
+  margin-right: 15%;
+}
+.ncTitle {
+  color: #19467b;
+}
+
+.ncFormContainer {
+  background-color: #ffffff;
+  padding-bottom: 2%;
+  border-radius: 8px;
+  margin-top: 5%;
+  margin-bottom: 5%;
+}
+
+.ncTextBox {
+  margin-left: 2%;
+  margin-right: 2%;
+}
+
+.ncDescription {
+  margin-left: 2%;
+}
+.ncUploadButton {
+  margin-left: 4%;
+}
+.ncSubmitButton {
+  margin-right: 4%;
+  float: right;
+}
```

Comment:
```suggestion
  .ncCarouselWrapper{
    border-radius: 4px;
    margin-left: 2%;
    margin-right: 2%;
    overflow: hidden;
  }
}
```

Reply:
okay, thank you for your advice, fixed the bug